pkg/db/client.go

```diff
@@ -13,7 +13,10 @@ import (
 	"strconv"
 )
 
-const FormatDateTime = "2006-01-02 15:04:05"
+const (
+	FormatDateTime           = "2006-01-02 15:04:05"
+	mysqlDuplicateEntryError = 1062
```

Comment:
instead of `mysqlDuplicateEntryError`, should we call it `mysqlDuplicateEntryErrorNumber`?